src/Parser/RestoDruid/Modules/Features/CastEfficiency.js

```diff
@@ -29,14 +32,16 @@ class CastEfficiency extends CoreCastEfficiency {
     {
       spell: SPELLS.IRONBARK,
       category: CastEfficiency.SPELL_CATEGORIES.COOLDOWNS,
-      getCooldown: haste => 90,
+      getCooldown: (haste, combatant) => combatant.hasTalent(SPELLS.STONEBARK_TALENT.id) ? 60 : 90,
```

Comment:
`haste` can be omitted as it's not used. `(_, combatant)` should be enough. I guess it's a question of personal preference, if you want to express it's not affected by haste.

Reply:
gottem